web/template/layout/mlpa-header.gohtml

```diff
@@ -22,41 +22,53 @@
-                <a class="govuk-tabs__tab" href="{{ prefix (printf "/lpa/%s/payments" .Lpa.UID) }}">
-                    Fees
-                </a>
-            </li>
-            <li class="govuk-tabs__list-item {{ if eq .TabName "documents" }}govuk-tabs__list-item--selected{{ end }}">
-                <a class="govuk-tabs__tab" href="{{ prefix (printf "/lpa/%s/documents" .Lpa.UID) }}">
-                    Documents
-                </a>
-            </li>
-            <li class="govuk-tabs__list-item {{ if eq .TabName "history" }}govuk-tabs__list-item--selected{{ end }}">
-                <a class="govuk-tabs__tab" href="#history">
-                    History
-                </a>
-            </li>
-        </ul>
+    <div class="case-management-row govuk-grid-row">
+        <div class="govuk-tabs govuk-grid-column-two-thirds">
+            <h2 class="govuk-tabs__title">
+                Contents
+            </h2>
+            <ul class="govuk-tabs__list">
+                <li class="govuk-tabs__list-item {{ if eq .TabName "summary" }}govuk-tabs__list-item--selected{{ end }}">
+                    <a class="govuk-tabs__tab" href="{{ prefix (printf "/lpa/%s" .Lpa.UID) }}">
+                        Summary
+                    </a>
+                </li>
+                <li class="govuk-tabs__list-item {{ if eq .TabName "details" }}govuk-tabs__list-item--selected{{ end }}">
+                    <a class="govuk-tabs__tab" href="#lpa-details">
+                        LPA details
+                    </a>
+                </li>
+                <li class="govuk-tabs__list-item {{ if eq .TabName "issues" }}govuk-tabs__list-item--selected{{ end }}">
+                    <a class="govuk-tabs__tab"  href="#issues">
+                        Issues
+                    </a>
+                </li>
+                <li class="govuk-tabs__list-item {{ if eq .TabName "fees" }}govuk-tabs__list-item--selected{{ end }}">
+                    <a class="govuk-tabs__tab" href="{{ prefix (printf "/lpa/%s/payments" .Lpa.UID) }}">
+                        Fees
+                    </a>
+                </li>
+                <li class="govuk-tabs__list-item {{ if eq .TabName "documents" }}govuk-tabs__list-item--selected{{ end }}">
+                    <a class="govuk-tabs__tab" href="{{ prefix (printf "/lpa/%s/documents" .Lpa.UID) }}">
+                        Documents
+                    </a>
+                </li>
+                <li class="govuk-tabs__list-item {{ if eq .TabName "history" }}govuk-tabs__list-item--selected{{ end }}">
+                    <a class="govuk-tabs__tab" href="#history">
+                        History
+                    </a>
+                </li>
+            </ul>
+        </div>
+
+        <div class="govuk-grid-column-one-quarter govuk-form-group">
+            <nav class="govuk-grid-column-one-quarter govuk-form-group" data-module="dropdown-menu">
```

Comment:
We should prefix `data-module` values with "app-" to clarify their scope (similarly there are "govuk-" and "moj-" modules):
```suggestion
            <nav class="govuk-grid-column-one-quarter govuk-form-group" data-module="app-dropdown-menu">
```